src/AddTodoForm.js

```diff
@@ -19,11 +19,13 @@ function AddTodoForm({onAddTodo}) {
 
 	return (
 		<>
-			<InputWithLabel
-				todoTitle={todoTitle}
-				handleTitleChange={handleTitleChange}
-			>Title</InputWithLabel>
-			<form onSubmit={handleAddTodo}>
+			<form onClick={handleAddTodo}>
```

Comment:
Are you using the correct handler prop in form?

Reply:
I changed the onClick handler to onSubmit. I assume this time it shouldn't have any bug :)